Makefile

```diff
@@ -428,12 +428,16 @@ DEFAULT_SETUP_PY_ENV := \
 	CXXFLAGS="$(CXXFLAGS) -I$(BUILD_STAGE)/python3/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include/$$(ls $(BUILD_STAGE)/python3/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include)" \
 	CPPFLAGS="$(CPPFLAGS) -I$(BUILD_STAGE)/python3/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include/$$(ls $(BUILD_STAGE)/python3/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include)"
 
+DEFAULT_RUST_FLAGS := \
+	SDKROOT="$(TARGET_SYSROOT)" \
+	PKG_CONFIG="$(RUST_TARGET)-pkg-config"
```

Comment:
Could the cargo build flags (`--release` and `--target`) be added to this as well, and then change the cargo call to this:
 ```mk
cargo build \
	$(DEFAULT_RUST_FLAGS)

Reply:
These flags have to be added beforehand. I have tested this before, and it did not work on my end. You can give it a shot and report back.

Reply:
alright, never mind then.